TABTestKit/Supporting Files/Info.plist

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Did this file not already exist?

Reply:
It exists on project example. I don't think it's a good idea to reference the Info.plist of the example project. It's two different projects.